packages/cli/src/index.ts

```diff
@@ -149,22 +142,28 @@ const main = async () => {
   }
 
   // Print update information, if available
-  if (notifier.update && notifier.update.latest !== pkg.version && isTTY) {
-    const { latest } = notifier.update;
-    console.log(
-      info(
-        `${chalk.black.bgCyan('UPDATE AVAILABLE')} ` +
-          `Run ${cmd(
-            await getUpdateCommand()
-          )} to install ${getTitleName()} CLI ${latest}`
-      )
-    );
+  if (isTTY && !process.env.NO_UPDATE_NOTIFIER) {
+    // Checks for available update and returns an instance
+    const latest = await updateNotifier({
+      distTag: isCanary ? 'canary' : 'latest',
+      pkg,
+    });
+    if (latest) {
+      console.log(
+        info(
+          `${chalk.black.bgCyan('UPDATE AVAILABLE')} ` +
+            `Run ${cmd(
+              await getUpdateCommand()
+            )} to install ${getTitleName()} CLI ${latest}`
+        )
+      );
 
-    console.log(
-      info(
-        `Changelog: https://github.com/vercel/vercel/releases/tag/vercel@${latest}`
-      )
-    );
+      console.log(
```

Comment:
We should be using `output.log()` here instead of `console.log()` otherwise we'll break the pipe-ability of commands (i.e. deployment URL goes to stdout for `vc deploy`).

I realize that this was the previous behavior as well, but since we're here we might as well fix it.

Reply:
Because of the `isTTY` check, this won't break piped output, but by convention and for future testing, we should still probably change these to use `output` in a follow-up PR.

Reply:
Can we update this in a follow-up PR?